tailwind.config.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
         seqblue: "#1B557A",
         seqorange: "#C74F13",
       },
+      fontSize: {
+        xs : ".7rem",
+      },
```

Comment:
I don't think we should change global tailwind sizes. The reason is that if we use openly available components like Flowbite they will look different with tailwinds global config changed. 

Also, I believe @yannickwurm's main concern was inconsistent and too many different font sizes on the page, so just make sure we use font sizes consistently and if there's some non-tailwind CSS tweaking font sizes via global CSS rules, address that instead